### ISSUE TITLE
'unexpected' responses on valid OAS3.0 yaml

### ISSUE THREAD

**Issue by tim-digitised** [kevinoid/swagger-spec-validator#45]
I've given your validator a run on my APIs and when running the CLI command, I get a lot of 'unexpected' responses on valid OAS3 yaml (validated in SwaggerHub.io)

For example, this is one of many unexpected responses I get:
`attribute paths.'/campaigns'(put).200.content is unexpected`

And the associated YAML for this is:
```
responses:
  '200':
    description: Success
    content:
      application/json:
        schema:
          $ref: '#/components/schemas/Campaign'
      application/xml:
        schema:
          $ref: '#/components/schemas/Campaign'
```

Not sure why your script is telling me the `content` attribute is unexpected when it's a valid OAS3 property. 

I guess the simple question is, does your app support OAS3?

Looking at the last two errors reported, I think it's not:
```
error: object instance has properties which are not allowed by the schema: ["components","openapi","servers"]
error: object has missing required properties (["swagger"])
```

**Comment by kevinoid** [comment]
Thanks for reporting the issue!

Unfortunately, this module doesn't support OAS3 yet.  It relies entirely on the [Swagger Validator Badge](https://github.com/swagger-api/validator-badge) hosted at https://online.swagger.io/validator to perform validation.  OAS3 support is being tracked in swagger-api/validator-badge#115.

I'll leave this issue open to track OAS3 support in this project, in case any changes need to be made for OAS3 support once `online.swagger.io` is updated.

**Comment by tim-digitised** [comment]
Hey mate, thanks for the quick response. I wouldn't really call this an 'issue' though as it's not quite supported yet. 

Maybe the only suggestion I could say is to add in your readme that it currently only supports 2.0. 
Reason for this is you link to the OAS spec page by swagger, but that starts off with 3.0 spec instead of 2.0 spec.

**Comment by kevinoid** [comment]
Good catch.  Thanks for pointing that out!

I've just pushed  be18028, which should help clarify a bit.  Let me know if you notice anywhere else where it is unclear.

**Comment by kevinoid** [comment]
Both `online.swagger.io` and `validator.swagger.io` now support OAS3!  I have just released `swagger-spec-validator@4.0.0` with support for the updated validator API (which requires `Content-Type` on every request).  If you are still having issues validating OAS3 documents, please reopen this issue.
